student-test/assignment1/Test_validate_username.c

```diff
@@ -17,6 +17,6 @@ void test_validate_my_username()
     /**
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
-     */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+     */ 
+	TEST_ASSERT_EQUAL_STRING_MESSAGE(my_username(), malloc_username_from_conf_file(),"fixed");
```

Comment:
@VisweshBaskaran when would `fixed` be printed in this case?  What cleanup is needed after `malloc_username_from_conf_file`?

Reply:
Should say the opposite instead of fixed as _MESSAGE takes the extra argument to print in case of failure

free needs to be called, but since I am not storing it in any variable how would I proceed?